be/src/storage/lake/schema_change.cpp

```diff
@@ -328,6 +328,60 @@ Status SchemaChangeHandler::do_process_alter_tablet(const TAlterTabletReqV2& req
     return Status::OK();
 }
 
+Status SchemaChangeHandler::process_update_tablet_meta(const TUpdateTabletMetaInfoReq& request) {
+    if (!request.__isset.txn_id) {
+        return Status::InternalError("txn_id not be set");
+    }
+    int64_t txn_id = request.txn_id;
+
+    for (const auto& tablet_meta_info : request.tabletMetaInfos) {
+        auto status = do_process_update_tablet_meta(tablet_meta_info, txn_id);
+        if (!status.ok()) {
+            return status;
+        }
+    }
+
+    return Status::OK();
+}
+
+Status SchemaChangeHandler ::do_process_update_tablet_meta(const TTabletMetaInfo& tablet_meta_info, int64_t txn_id) {
+    MonotonicStopWatch timer;
+    timer.start();
+    LOG(INFO) << "begin to update tablet. tablet: " << tablet_meta_info.tablet_id
+              << "update meta type:" << tablet_meta_info.meta_type;
+
+    auto tablet_id = tablet_meta_info.tablet_id;
+    ASSIGN_OR_RETURN(auto tablet, _tablet_manager->get_tablet(tablet_id));
+
+    if (tablet.get_txn_log(txn_id).ok()) {
+        LOG(INFO) << "the update tablet meta task has been processed, txn_id:" << txn_id;
+        return Status::OK();
+    }
```

Comment:
I think this check is unnecessary since it introduces extra I/O, while in most cases the txn log does not exist.

Reply:
This is for handling duplicate `alter_meta_task` in some abnormal situations，I'm not sure whether there will be something wrong if that happens, writing the same txn_log.
and  `alter_meta_task` does not occur very frequently.

Reply:
Concurrent or overwriting writes are acceptable as long as the txn log content remains the same. Besides, the txn log read here can't actually prevent concurrent or overwriting writes in abnormal cases, since the read and write below are not atomic.